cmd/install/addon/cmd.go

```diff
@@ -197,72 +196,51 @@ func run(cmd *cobra.Command, argv []string) {
 		parameters.Each(func(param *cmv1.AddOnParameter) bool {
 			var val string
 			var hasVal bool
+			var optionStrings []string
+
+			options, _ := param.GetOptions()
+			for _, opt := range options {
+				optionStrings = append(optionStrings, opt.Value())
```

Comment:
I think it would be better to create a list of `option.Name()`. The value is arbitrary, where as the name has some context behind it